system/jlib/jexcept.cpp

```diff
@@ -1284,6 +1286,7 @@ void  jlib_decl disableSEHtoExceptionMapping()
     signal(SIGBUS, SIG_DFL);
     signal(SIGILL, SIG_DFL);
     signal(SIGFPE, SIG_DFL);
+    signal(SIGABRT, SIG_DFL);
```

Comment:
I find some other thing and I will update this PR soon and I think it will need a one more final check.